elliott

```diff
@@ -348,11 +353,17 @@ advisory.
 @click.option("--id", type=int, metavar='BUGID',
               multiple=True, required=False,
               help="Bugzilla IDs to add, conflicts with --auto [MULTIPLE]")
+@click.option("--old",
+              required=False,
+              help="Older payload to compare against")
+@click.option("--new",
+              required=False,
```

Comment:
I think your idea on slack is a good one. Some kind of `--between A B` thing to see what's in `B` but not `A`. Don't even need to include fancy `...`s in there. Just set the `nargs` to 2 (I'm pretty sure click does that).